src/transformers/models/wav2vec2/processing_wav2vec2.py

```diff
@@ -78,8 +78,10 @@ def __call__(self, *args, **kwargs):
         if "raw_speech" in kwargs:
             warnings.warn("Using `raw_speech` as a keyword argument is deprecated. Use `audio` instead.")
             audio = kwargs.pop("raw_speech")
+            sampling_rate = kwargs.pop("sampling_rate", None)
         else:
             audio = kwargs.pop("audio", None)
+            sampling_rate = kwargs.pop("sampling_rate", None)
```

Comment:
Since it's there in both path, you should just put it once outside of the if ;-)

Reply:
Whoops 😅 you're entirely correct! Resolved in [5568d32](https://github.com/huggingface/transformers/pull/20022/commits/5568d32eb0823a029117c0067e33cd7093f2fe9d)